core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.mainpage.html

```diff
@@ -43,6 +43,169 @@
+           class="oppia-editor-navbar-tab-anchor protractor-test-preview-tab">
+          <i class="material-icons">&#xE037;</i>
+        </a>
+      </li>
+      <li ng-class="{'active': getActiveTabName() === 'settings'}">
+        <a href="#" uib-tooltip="Settings" tooltip-placement="bottom" ng-click="selectSettingsTab()"
+           class="oppia-editor-navbar-tab-anchor protractor-test-settings-tab">
+          <i class="material-icons">&#xE8B8;</i>
+        </a>
+      </li>
+      <li ng-if="userIsLoggedIn" ng-class="{'active': getActiveTabName() === 'stats'}">
+        <a href="#" uib-tooltip="Statistics" tooltip-placement="bottom" ng-click="selectStatsTab()"
+           class="oppia-editor-navbar-tab-anchor protractor-test-stats-tab">
+          <i class="material-icons">&#xE24B;</i>
+        </a>
+      </li>
+      <li ng-if="isImprovementsTabEnabled()" ng-class="{'active': getActiveTabName() === 'improvements'}" ng-click="selectImprovementsTab()">
+        <a href="#" uib-tooltip="Improvements" tooltip-placement="bottom"
+           class="oppia-editor-navbar-tab-anchor protractor-test-improvements-tab">
+          <i class="material-icons">trending_up</i>
+        </a>
+      </li>
+      <li ng-class="{'active': getActiveTabName() === 'history'}">
+        <a href="#" uib-tooltip="History" tooltip-placement="bottom" ng-click="selectHistoryTab()"
+           disabled="ExplorationRightsService.isCloned()"
+           class="oppia-editor-navbar-tab-anchor protractor-test-history-tab">
+          <i class="material-icons">&#xE192;</i>
+        </a>
+      </li>
+      <li  ng-if="isFeedbackTabEnabled()" ng-class="{'active': getActiveTabName() === 'feedback'}" ng-click="selectFeedbackTab()">
+        <a href="#" uib-tooltip="Feedback" tooltip-placement="bottom"
+           class="oppia-editor-navbar-tab-anchor protractor-test-feedback-tab">
+          <i class="material-icons">&#xE87F;</i>
+        </a>
+        <div ng-show="getOpenThreadsCount()" class="oppia-exploration-open-threads-indicator oppia-exploration-open-threads-color">
+          <span class="oppia-exploration-open-threads-count">
+              <[getOpenThreadsCount()]>&nbsp;open
+          </span>
+        </div>
+      </li>
+        <!-- <div ng-if="userIsLoggedIn" class="nav navbar-nav oppia-navbar-nav oppia-help-dropdown pull-right ng-cloak"> -->
+      <li ng-if="userIsLoggedIn" uib-dropdown class="uib-dropdown uib-popover ng-cloak" popover-is-open="popoverControlObject.postTutorialHelpPopoverIsShown"
+          popover-placement="bottom" popover-trigger="'none'"
+          uib-popover="To get help in the future, click here!">
+        <a href="#" uib-tooltip="Help" tooltip-placement="bottom"
+           class="oppia-editor-navbar-tab-anchor" ng-click="showUserHelpModal()">
+          <i class="material-icons">&#xE887;</i>
+        </a>
+      </li>
+            <!-- </div> -->
+    </ul>
+  </div>
```

Comment:
Could you separate this into a directive (similar to `<editor-navbar-breadcrumb></editor-navbar-breadcrumb>`)?

Reply:
I have made into a directive.